internal/graphql/model/operator_helper.go

```diff
@@ -103,11 +114,14 @@ func GetBrokerInfo(mesheryclient operatorClient.Interface, mesheryKubeClient *me
 	brokerVersion := ""
 	if err == nil {
 		brokerVersion = imageVersionExtractUtil(statefulSet.Spec.Template, "nats")
-	}
-	if err == nil {
-		status := fmt.Sprintf("%s %s", StatusConnected, broker.Status.Endpoint.External)
-		if brokerConn.Info() != brokerpkg.NotConnected {
-			brokerStatus.Status = Status(status)
+		externalIP := strings.Split(broker.Status.Endpoint.External, ":")[0]
+		endpoint := fmt.Sprintf("http://%s%s/%s", externalIP, ":8222", "connz")
+
+		if connectivityTest(models.MesheryServerBrokerConnection, endpoint) {
```

Comment:
Looks like we only have logic for `StatusConnected` and/or its negation. But we also have `Deployed`, `NotDeployed` etc., right?